src/makielayout/interactions.jl

```diff
@@ -298,6 +298,28 @@ function process_interaction(dp::DragPan, event::MouseEvent, ax)
     timed_ticklabelspace_reset(ax, dp.reset_timer, dp.prev_xticklabelspace, dp.prev_yticklabelspace, dp.reset_delay)
 
     tlimits[] = FRect(Vec2f0(xori, yori), widths(tlimits[]))
-           
+
     return nothing
 end
+
+############################################################################
+#             Interactions that return values when activated               #
+############################################################################
+function AbstractPlotting.select_rectangle(ax::MakieLayout.LAxis; kwargs...)
+    deactivate_interaction!(ax, :rectanglezoom)
```

Comment:
I don't think interactions should deactivate others by default. I'm not sure what the best solution is, but there could always be other interactions that are enabled, which could conflict. So the person who enables one interaction needs some knowledge about what they disable instead, I think.

Reply:
Rectanglezoom could also theoretically be set to a different non-conflicting mouse button, etc.

Reply:
> I don't think interactions should deactivate others by default.

It doesn't have to be by default, it can be a keyword option to `select_rectangle`. Of course I agree that the developer that uses `select_rectangle` (or whatever interaction) should be aware of what interactions are present.

> Rectanglezoom could also theoretically be set to a different non-conflicting mouse button, etc.

I disagree, because the current amount of keys/buttons taken by the default interaction on `LAxis` is simply too much, leaving no sensible options. Many people have mouses with only two buttons (anything else besides a mouse button doesn't feel intuitive for `interactive_rectangle`). At the moment by default both left and right clicks are taken. Besides, there is already the scroll wheel available for zooming, this (in my eyes) only enhances the argument that left click should be allowed to be rebinded, given that zooming possibility isn't actually lost.

Reply:
I now realize that a sensible option would be the case of having to press Ctrl first (but I don't know how to do the proper checks for this)

Reply:
on the other hand one can also consider letting Left mouse button be free, the option to be used by front-end interactive applications, and make the zoom rectangle require control